cmd/virt-handler/virt-handler.go

```diff
@@ -305,14 +305,28 @@ func (app *virtHandlerApp) Run() {
 
 	stop := make(chan struct{})
 	defer close(stop)
-	var capabilities *libvirtxml.Caps
```

Comment:
Please consider adding a few words to the commit message about the two major advantages we get from this change:
1. Improved testability of NodeLabeller 
2. Decoupling of VMController from NodeLabeller.

Reply:
> I don't follow this part, could you please elaborate?

Logging the failure to obtain TSC frequency values for every `prepareLabels` run is redundant. It is possible to move the log message somewhere else (e.g. node-labeller `Run`), and upon further consideration I'll do just that.

> I think we could add options_test.go and test at least capabilitiesToTopology in order to verify that the change introduced by this PR hasn't caused a regression.

I've added it in a separate commit, the construction of the capabilities for the test is a bit complex but I preferred it over adding a testdata file or declaring a variable that contains the relevant data in XML form, would like feedback on that.

> Please consider adding a few words to the commit message about the two major advantages we get from this change:
> 1. Improved testability of NodeLabeller
> 2. Decoupling of VMController from NodeLabeller.

Done.